internal/util/get/get.go

```diff
@@ -95,6 +95,10 @@ func (c Command) Run(ctx context.Context) error {
 	}
 
 	if err = c.fetchPackages(ctx, p); err != nil {
+		rmErr := os.RemoveAll(c.Destination)
```

Comment:
Clean up should happen for any error after line 68.

Reply:
Done.